huaweicloud/resource_huaweicloud_fgs_function_v2.go

```diff
@@ -125,6 +118,122 @@ func resourceFgsFunctionV2() *schema.Resource {
 					}
 				},
 			},
+			"depend_list": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"initializer_handler": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"initializer_timeout": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"enterprise_project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"func_vpc": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"vpc_id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"subnet_id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"vpc_name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"subnet_name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"cidr": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"gateway": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"mount_config": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"mount_user": {
```

Comment:
it's unnecessary to wrap user_id and user_group_id into mount_user

Reply:
fixed